pkg/yurtadm/cmd/join/phases/joinnode.go

```diff
@@ -70,6 +82,11 @@ func runJoinNode(c workflow.RunData) error {
 	if err != nil {
 		return err
 	}
+
+	if err := addLvsStaticPodYaml(data, filepath.Join(constants.KubernetesDir, constants.ManifestsSubDirName)); err != nil {
```

Comment:
why need to install kube-lvscare on the edge node? I think we can use the load balancer ability in yurthub.

Reply:
kube-lvscare provide load balancing for the previous VIP.

Reply:
This part of the code comes from the source code of `sealer join`.

Reply:
Yurthub already has load balancing capability, it seems that VIP is really not needed. `kubedm join` just needs to specify the IP address of any master, because later, you access apiserver through yurthub.

Reply:
solved.